database/factories/MovieWatchlistFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Model;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class MovieWatchlistFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Model::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            //
```

Comment:
Try to add values to the columns 😄 

Check the DB structure and try to create some values here, as this will help with testing.